catkin_ws/src/bringup/launch/bringup.launch

```diff
@@ -25,6 +23,11 @@
 	<include file="$(find propulsion)/launch/propulsion.launch">
 		<arg name="sim" value="$(arg sim)" />
 	</include>
+	
```

Comment:
the only problem is that the status check for hydrophones depends on hydrophones.launch params. So, if you put sensors before state_estimation, hydrophones status is going to be stuck in a loop until hydrophones.launch is launched

Reply:
i can change that pretty easily